ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/ButtonOrder.kt

```diff
@@ -0,0 +1,52 @@
+package com.braintreepayments.api.shopperinsights
+
+/**
+ * The order or ranking in which payment buttons appear.
+ */
+enum class ButtonOrder(internal val stringValue: String) {
+
+    /**
+     * First place
+     */
+    FIRST("1"),
```

Comment:
Does it matter if we are sending these numbers as strings vs Int on iOS? Does it come through in FPTI the same way?

Reply:
I think we may need to do it this way because of the last enum being "other".